### ISSUE TITLE
Search by tag, project

### ISSUE THREAD

**Issue by rtyshyk** [moranje/alfred-workflow-todoist#20]
Thanks for the workflow!
Could you please add search by tag/project in the same was as it is implemented in todoist search?

Thanks,
Roman

**Comment by moranje** [comment]
Hi Roman,

Thanks for opening a ticket. I do intend to implement a better search
feature in the future that allows searching by tag, priority an project.
However it will likely take some time before I'll be able to allocate some
open source time again. Keep an eye on this issue.

Thanks

-- 
Martien Oranje

Op 28 juni 2017 bij 21:29:59, Roman (notifications@github.com) schreef:

Thanks for the workflow!
Could you please add search by tag/project in the same was as it is
implemented in todoist search?

Thanks,
Roman

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/moranje/alfred-workflow-todoist/issues/20>, or mute the
thread

**Comment by mikedett91** [comment]
I know this has been open for a bit but was wondering if you had been able to look into this functionality more, I feel like that one feature would help round out an already excellent workflow!

Being able to search by due-date would be the first thing that I personally would desire, or a way to filter out recurring tasks that you've completed for the day, etc

**Comment by moranje** [comment]
Yes, this will be fixed in the rewritten version of this workflow.

Martien

Op di 14 nov. 2017 22:10 schreef mikedett91 <notifications@github.com>:

**Comment by moranje** [comment]
Still intend to (re)implement this

**Comment by moranje** [comment]
Should be fixed in the v6.0.0-alpha.3. It should be stable enough to use as your daily driver, I now have tests for 80% of the code base. 

If you want to keep receiving updates set `todo:settings pre_releases true`. I still need to update the docs so not all the changes are documented yet.

Martien
